nemo/collections/cv/modules/video_preprocessing.py

```diff
@@ -0,0 +1,109 @@
+import torch
+import torchvision
+from torch import nn
+
+from nemo.collections.cv.modules import Permute
+from nemo.core.classes import NeuralModule, typecheck
+
+
+class VideoPreprocessor(NeuralModule):
+
+    """ Lip Reading Video Pre-processing
```

Comment:
What's specific to lip reading in this class?

Reply:
changed to Video Pre-processing